index.php

```diff
@@ -0,0 +1,270 @@
+                            </li>
+                            <li>
+                                <a href="#formation">PARCOURS
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#experience">EXPERIENCE</a>
+                            </li>
+                            <li>
+                                <a href="#competence">COMPÉTENCES</a>
+                            </li>
+                            <li>
+                                <a href="#portfolio">PORTFOLIO</a>
+                            </li>
+                            <li>
+                                <a href="#contact">CONTACT</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <button id="hamburger-button">&#9776;</button>
+                <div id="hamburger-sidebar">
+                    <div id="hamburger-sidebar-header"></div>
+                    <div id="hamburger-sidebar-body"></div>
+                </div>
+                <div id="hamburger-overlay"></div>
+            </div>
+        </header>
+        <!-- NAV END ***************************************** -->
+
+        <!-- SLIDER ***************************************** -->
+
+        <section id="slider">
+            <div>
+                <img src="public/images/slider.png" alt="Background coucher de soleil">
+            </div>
+            <div class="background"></div>
+            <div class="pres row align-items-center headline">
+                <h1>
+                    <span class="tagline">Frédéric Vanmarcke</span>
+                    <br />
+                    <span class="int tagline">Développeur - Intégrateur Web & Web Mobile à Rambouillet</span>
+                   
+                </h1>
+            </div>
+            <a id="toDown" title="Go to down" href="#me" class="fa fa-chevron-down punchline">
+            </a>
+        </section>
+        <!-- SLIDER END ***************************************** -->
+        
+        <?php
+        // ABOUT ME *****************************************
+
+         include 'public/view/aboutMe.twig';
+
+        //  TRAINING *****************************************
+        
+         include 'public/view/training.twig';
+
+        //  EXPERIENCE *****************************************
+       
+         include 'public/view/experience.twig';
+
+        //  SKILL  *****************************************
+    
+         include 'public/view/skill.twig';
+
+        //  PORTFOLIO  *****************************************
+     
+         include 'public/view/portFolio.twig';
+
+        //  HOBBIES  *****************************************
+        
+         include 'public/view/hobbies.twig';
+        ?>
+        <!-- CONTACT  ***************************************** -->
+       
+        <section id="contact">
```

Comment:
Pourquoi ne pas avoir fait d'include de contact ?

Reply:
Contact sera mis en "include" lors de l'issue "Add file required for routing" avec le déplacement de "EmailProcessing.php"